.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Would you mind splitting this to two commits? One for Maven wrapper and one for SCS fix.